docs/home/whats-new-2-15.md

```diff
@@ -1,94 +1,103 @@
+<!-- RUN-7085/RUN-9480 Installation - Cluster wizard Improvements -->
+* Improved the *Cluster Wizard* form for adding new clusters to your system. We have eliminated the need for a values file and converted it to a helm command. Cluster configurations are preserved during upgrade and are done using the `runaiconfig` file and create a separation between installation-related flags and cluster customization flags. For more information, see [Customize cluster installation.](##customize-cluster-install.md)
 
-<!-- RUN-10602/RUN-10603 GPU Memory Request & Limit-->
+<!-- ADDLINK RUN-9924/RUN-9925  Granular GPU compute time-slicing / Strict GPU compute time-slicing -->
+* Added the ability to configure strict GPU compute time slicing. This gives workloads the exact GPU compute portion based on the requested GPU fraction (GPU Memory Fraction). This creates complete transparency and predictability of the amount of resources (Compute, Memory, etc.) a workload will get from a GPU. For more information, see [GPU Time Slicing](##gpu-time-slicing).
 
-<!-- RUN-10619 Email mechanism -->
+<!-- RUN-10862/RUN-10863 Department as a workspace phase 2 - scope in credentials -->
+* Improved *Credentials* creation. Now, a Run:ai scope can be added to credentials. For more information, see [Credentials](../admin/admin-ui-setup/credentials-setup.md).
 
-<!-- RUN-10622 Policy blocks workloads that attempt to store data on the node-->
+<!-- RUN-10271/RUN-10321 Mark environment for workload type-->
+* Added support for workload types when creating a new or editing and existing environment. Select from `single-node` or `multi-node (distributed)` workloads. The environment is available only on feature forms which are relevant to the workload type selected.
 
-<!-- RUN-10639/RUN-11389 Researcher Service Refactoring - Step 1 \(UI only\)-->
+<!-- RUN-10639/RUN-11389 - Researcher Service Refactoring -->
+* Improved configuration for authentication in the cluster. Configuring the Kubernetes cluster API server for authentication is no longer needed. This applies to environments where end users are using the UI only and not the CLI.
 
-<!-- RUN-10802 Quota management dashboard post-release -->
+<!-- RUN-12505/RUN-12506 - Support Kubeflow notebooks for scheduling/orchestration -->
+* Improved support for KubeFlow Notebooks. Now Run:ai supports scheduling of Kubeflow notebook CRDs with fractional GPUs. Kubeflow notebooks are identified automatically and use a special icon in the *Jobs* UI.
 
-<!-- RUN-10862/RUN-10863 Department as a workspace asset creation scope - phase 2-->
+<!-- RUN-10251/RUN-10252 - Block over-subscription of quota by Projects/Departments- -->
+* Improved quota settings by providing the ability to block over-subscription of quota by *Projects* or *Departments*. For more information, see [Over quota blocking](../Researcher/scheduling/the-runai-scheduler.md#limit-quota-over-or-under-subscription).
```

Comment:
I would not say that we improved the settings. This is the how.
The what is that the control over how over-quota is managed was improved by adding the ability to configure...